# robot : SHIV34 : 1045x700 : 522.5x350 ( tinh tu tam robot )
dx_robot : 0.53 
dy_robot : 0.35


# vi tri lidar ( tinh tu tam robot )
dx_lidar : 0.3625
dy_lidar : 0.2625

# zone 1 ( tinh tu head robot)
# dx1 : 0.6
# zone 2
# dx2 : 1.2
# zone 3 - dang cho ke hang 
# dx3 : 0.3

# ke hang 
# dx_shelves : 0.6
dy_shelves : 0.60 

# AHEAD ( tinh tu head robot)
dx1_head : 0.6 # zone 1 
dx2_head : 0.8 # zone 2 :  ke hang 
dx3_head : 1.3 # zone 3 

# BEHIND ( tinh tu behind robot)
dx1_behind : 0.2 # zone 1 
dx2_behind : 0.6 # zone 2
dx3_behind : 1.2 # zone 3

# 0 : only 2D , 1 : 2D+3D 
zone_2d_3d : 1
